docs/instrument-traces/js-app.md

```diff
@@ -0,0 +1,145 @@
+# Javascript App
+
+### Auto-instrument sample Javascript app for traces
+
+In this tutorial we will go through the steps to auto instrument a Javascript app to send traces to Siglens.
+
+### Prerequisites
+- Siglens instance should be running on localhost with ingest port-4318. To do so you need to change the ingest port of Siglens to `4318` in `server.yaml`
+- Javascript app (refer the documentation below if you don't have the setup for javascript app)
+
+### Set up for Javascript application
+
+Given below are the instructions for setting up a sample Javascript application:
+
+Note:                                                                                
+To begin with the setup you must have Node.js installed locally. To check if node is installed you can run the following command:
```

Comment:
Since having Node.js is a prerequisite, it should be mentioned in the Prerequisites section. It might also be good to link to some other site with installation instructions in case they don't have it installed.